### PR TITLE
Update actionbar.simba

### PR DESCRIPTION
**Pull request by ThomasSRL**
Added a few things, hope I didn't break anything. So please be sure to check if everything is updated properly...

The 3 last functions can be deleted. But could ruin a couple of scripts.

### CODE REVIEW

**Comment by ThomasSRL** [comment]
Read the notes on commit before testing.

File: lib/interfaces/actionbar.simba [hunk 1]
```diff
@@ -1298,37 +1324,94 @@ procedure TRSActionBar.mouseIcon(icon, button: integer = MOUSE_LEFT);
...
+  if not inRange(icon, 1, 3) then
+    exit();
+
+  case icon of
+    1: color := 4884648;  //Healing
+    2: color := 3442349;  //Adrenaline
+    3: color := 11751792; //Prayer
+  end;
+
+  result := countColorTolerance(color, iconBoxes()[icon - 1], 100) > 165;
```
Comment (ThomasSRL) [inline lib/interfaces/actionbar.simba]: result :+1: = countColorTolerance(color, iconBoxes()[icon - 1], 90) > 160;

Works better. Had a little error.

**Comment by TheTS** [comment]
You should really make a thread to discuss these things if you're going to change this much. If you're going down this route, then it would be better to make the old functions call the new one to avoid too much redundant code.

```pascal
function TRSActionBar.setQuickPrayer(enabled: boolean = true): boolean;
begin
  result := self.__setIcon(AB_BAR_PRAYER, enabled);
end;
```

**Comment by ThomasSRL** [comment]
Yes, planned On doing that.
Ill post a thread whenever I got some spare time!

**Pull request closed without merging**